### ISSUE TITLE
Specific Content-Type support for PresignedUrlRequest 

### ISSUE THREAD

**Issue by starr749** [EMCECS/ecs-object-client-java#6]
There is currently no way to specify Content-Type when using Pre-Signed URLs.

If one is specified, it throws the error:

 
```
 SignatureDoesNotMatch
The request signature we calculated does not match the signature you provided. Check your Secret Access Key and signing method. For more information, see REST Authentication and SOAP Authentication for details.
```
I assume this is because no Content-Type was set when the URL was generated. But there is support for this in the ECS S3 Java SDK.

There is support for it in the AWS S3 SDK explained [here](http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/s3/model/GeneratePresignedUrlRequest.html) 

Specifically _**withContentType(String contentType)**_

It would be great is this functionality was added, as we could then lock down the Content-Type with our pre-signed URLs.

**Comment by ben-schumacher** [comment]
Hi @starr749,
Thank you for the suggestion.  We're working to get this added into the SDK as it's a very real use case in the S3 API.  In the meantime, your testing should succeed by **not** specifying a Content-Type header when using the pre-signed URL.  Will keep you posted!

**Comment by twincitiesguy** [comment]
Fixed in v3.0.5
